README.md

```diff
@@ -62,6 +63,7 @@ The _"non-endorsed"_ status means that the plugin is not endorsed by the origina
 | [**flutter_app_badger_tizen**](packages/flutter_app_badger) | 4.0 | ✔️ | ✔️ | ❌ | ❌ | API not supported |
 | [**flutter_secure_storage_tizen**](packages/flutter_secure_storage) | 4.0 | ✔️ | ✔️ | ✔️ | ✔️ |
 | [**flutter_tts_tizen**](packages/flutter_tts) | 4.0 | ✔️ | ✔️ | ✔️ | ✔️ |
+| [**flutter_webrtc_tizen**](packages/flutter_webrtc) | 6.0 | ❌ | ❌ | ✔️ | ❌ | No camera |
```

Comment:
Can I test this plugin on a mobile device (which has a camera attached)? If so, do you have a plan to support emulators (to just make debugging easier)?

Reply:
If mobile device with a camera, I think plugin can run on this device.
I haven't tested it on the emulators, I will try.

Reply:
I run webrtc demo on tizen 6.0 mobile emulator,  emulator not support camera, but it can show remote video.

Reply:
Thank you. If you want to test camera on mobile emulator, you can use a laptop with a built-in camera or connect a USB camera to your desktop. We usually test the camera_tizen plugin in that way.

Reply:
I plugged a camera into the PC and camera app can show video successful. but flutter webrtc can't get preview callback.

Reply:
I check the flutter camera plugin source code, I found plugin example set the format is CameraPixelFormat::kJPEG.
but webrtc can't  JPEG type when try get all supported format:
```cpp
camera_foreach_supported_preview_format
```
```bash
	Line  22: I/TizenWebRTC(15642): (device_info_tizen.cc:108): CreateCapabilityMap called for device CAMERA_DEVICE_CAMERA0
	Line  23: I/TizenWebRTC(15642): (device_info_tizen.cc:193): Camera capability, width: 176 height: 144 type: VideoType::kYUY2 fps: 30
	Line  24: I/TizenWebRTC(15642): (device_info_tizen.cc:193): Camera capability, width: 320 height: 240 type: VideoType::kYUY2 fps: 30
	Line  25: I/TizenWebRTC(15642): (device_info_tizen.cc:193): Camera capability, width: 352 height: 288 type: VideoType::kYUY2 fps: 30
	Line  26: I/TizenWebRTC(15642): (device_info_tizen.cc:193): Camera capability, width: 640 height: 480 type: VideoType::kYUY2 fps: 30
	Line  27: I/TizenWebRTC(15642): (device_info_tizen.cc:193): Camera capability, width: 176 height: 144 type: VideoType::kYV12 fps: 30
	Line  28: I/TizenWebRTC(15642): (device_info_tizen.cc:193): Camera capability, width: 320 height: 240 type: VideoType::kYV12 fps: 30
	Line  29: I/TizenWebRTC(15642): (device_info_tizen.cc:193): Camera capability, width: 352 height: 288 type: VideoType::kYV12 fps: 30
	Line  30: I/TizenWebRTC(15642): (device_info_tizen.cc:193): Camera capability, width: 640 height: 480 type: VideoType::kYV12 fps: 30
```